### PR TITLE
add code of conduct

### PR DESCRIPTION
**Pull request by sbtaylor15**
Signed-off-by: Steve Taylor <steve@deployhub.com>

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct v0.5.0
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
```
Comment (markjacksonfishing) [inline CODE_OF_CONDUCT.md]: Should be `we as,`

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+contributors and maintainers pledge to making participation in our project and
```
Comment (markjacksonfishing) [inline CODE_OF_CONDUCT.md]: Should be `contributors and maintainers,`

Comment (markjacksonfishing) [inline CODE_OF_CONDUCT.md]: `making` should be `make`

**Review by markjacksonfishing — CHANGES_REQUESTED** [review]
Minor grammar changes

**Review by markjacksonfishing — APPROVED** [review]

**Review by TracyRagan — APPROVED** [review]
All good

**Pull request merged**